### DISCUSSION TITLE
👋 Welcome! 👉 pH7Builder Discussions 🎉

### DISCUSSION THREAD

**Discussion by pH-7** [pH7Software/pH7-Social-Dating-CMS#814] [General]
## 👋 Welcome!
  
pH7Builder (pH7 Social Dating CMS) is now using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Have a look to the clear and good 👉 [GitHub Community Guidelines](https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-community-guideline) if you are not sure what and what not to post 🤗

**Comment by Undefined-Variables** [comment]
Hey everyone! @pH-7 great idea! Somewhere to talk about the software... Long over due!

**Reply by pH-7** [reply]
Glad you like the idea @Undefined-Variables! I've been extremely busy lately, but hopefully, I will be back on track by now :) 
Anyway, hope the folks out there using the software will use this place to speak about things that aren't related to problems and issues from the software. Could also be "How to build a dating website", etc.

**Comment by Undefined-Variables** [comment]
There are a lot of open issues, I will answer a few now, I have been so busy lately but have some time now...
